src/cli/cli.cpp

```diff
@@ -49,14 +49,13 @@ const Menu_t CLI_menu[] =
     {0, nullptr, nullptr}
 };
 
-char userInput[SF_CLI_MAX_CMD_LEN];
+char inputBuffer[SF_CLI_MAX_CMD_LEN];
+
 
 STATES_e CLI_nextState;
 
 void CLI::init(void) 
 {
-    VERS_printBanner();
-
```

Comment:
Why is this being removed?

Reply:
Something got messed up in the merge, i'll have a look through it now

Reply:
Fixed